CHANGELOG.md

```diff
@@ -19,3 +19,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Integrate support in Helm resources for users with and without WSO2 subscriptions
 
 [v5.8.0.2]: https://github.com/wso2/kubernetes-is/compare/v5.8.0.1...v5.8.0.2
+
+## [V5.8.0.3] - 2019-08-15
```

Comment:
@Savidude let's keep this as `Unreleased` as in the example. We can set the date to the specific release date when we merge this branch with the `master` branch.

Reply:
Changed https://github.com/wso2/kubernetes-is/pull/159/commits/2d0ceb62519ef48cbdcb9f3be7ebcc1b1f911332